deploy/manifests/configmap.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: topolvm-config
+  namespace: topolvm-system
+data:
+  scheduler.yaml: |
+    listen: "localhost:9251"
```

Comment:
Do you know why this was changed from the default (`:8000`)?

Reply:
I don't know.
I think the default value should be 9251.